.pre-commit-config.yaml

```diff
@@ -3,20 +3,27 @@ repos:
     rev: v0.19.1
     hooks:
       - id: stylua-github
-
+        fail_fast: true
```

Comment:
The steps will interfere with each other and cause spurious subsequent failures.